tetrateci/create_istio_release.sh

```diff
@@ -71,6 +71,12 @@ cp -r ../istio .
 # export IMAGE_VERSION=$(curl https://raw.githubusercontent.com/istio/test-infra/master/prow/config/jobs/release-builder.yaml | grep "image: gcr.io" | head -n 1 | cut -d: -f3)
 # make shell TODO: https://github.com/tetratelabs/getistio/issues/82
 
+# Enable CGO flag"
+
+if [[ ${TAG} =~ "fips" ]]; then
+  text="if [[ "\${GOARCH}" == "amd64" ]]; then export CGO_ENABLED=1; else export CGO_ENABLED=0; fi"
+  sed -i '52s/.*/'"$text"'/g' istio/common/scripts/gobuild.sh
```

Comment:
it was manual change to `gobuild.sh` in 1.7 at least?

but since 1.10 release, upstream starts to allow set `CGO_ENABLE=1` from the caller.
https://sourcegraph.com/github.com/istio/istio/-/blob/common/scripts/gobuild.sh?L52&subtree=true
that would mean adding the env var in `manifest.docker.yaml` generation, I guess? Not sure.

I think it's fine for now. we could log an issue as backlog to track the potential improvement in future.

Reply:
sure, will do that.
can you approve it so we can give it a try.